_site/index.html

```diff
@@ -0,0 +1,205 @@
+</li>
+<li>
+<p><strong><a href="https://docs.rs/wasmtime-wasi/latest/wasmtime_wasi/">WASI</a></strong>. Wasmtime supports a rich set of APIs for interacting with the host
+environment through the <a href="https://wasi.dev">WASI standard</a>.</p>
+</li>
+<li>
+<p><strong><a href="https://docs.wasmtime.dev/stability-wasm-proposals-support.html">Standards Compliant</a></strong>. Wasmtime passes the <a href="https://github.com/WebAssembly/testsuite">official WebAssembly test
+suite</a>, implements the <a href="https://github.com/WebAssembly/wasm-c-api">official C
+API of wasm</a>, and implements
+<a href="https://github.com/WebAssembly/proposals">future proposals to WebAssembly</a> as
+well. Wasmtime developers are intimately engaged with the WebAssembly
+standards process all along the way too.</p>
+</li>
+</ul>
+</div>
+</section>
+<section>
+<div class="container w-container">
+<h2>Language Support</h2>
+<p>You can use Wasmtime from a variety of different languages through embeddings of
+the implementation:</p>
+<ul>
+<li><strong><a href="https://bytecodealliance.github.io/wasmtime/lang-rust.html">Rust</a></strong> - the <a href="https://crates.io/crates/wasmtime"><code>wasmtime</code> crate</a></li>
+<li><strong><a href="https://bytecodealliance.github.io/wasmtime/examples-c-embed.html">C</a></strong> - the <a href="https://bytecodealliance.github.io/wasmtime/c-api/"><code>wasm.h</code>, <code>wasi.h</code>, and <code>wasmtime.h</code> headers</a>, <a href="crates/c-api/CMakeLists.txt">CMake</a> or <a href="https://conan.io/center/wasmtime"><code>wasmtime</code> Conan package</a></li>
+<li><strong><a href="https://github.com/bytecodealliance/wasmtime-cpp">C++</a></strong> - the <a href="https://github.com/bytecodealliance/wasmtime-cpp"><code>wasmtime-cpp</code> repository</a> or use <a href="https://conan.io/center/wasmtime-cpp"><code>wasmtime-cpp</code> Conan package</a></li>
+<li><strong><a href="https://bytecodealliance.github.io/wasmtime/lang-python.html">Python</a></strong> - the <a href="https://pypi.org/project/wasmtime/"><code>wasmtime</code> PyPI package</a></li>
+<li><strong><a href="https://bytecodealliance.github.io/wasmtime/lang-dotnet.html">.NET</a></strong> - the <a href="https://www.nuget.org/packages/Wasmtime"><code>Wasmtime</code> NuGet package</a></li>
+<li><strong><a href="https://bytecodealliance.github.io/wasmtime/lang-go.html">Go</a></strong> - the <a href="https://pkg.go.dev/github.com/bytecodealliance/wasmtime-go"><code>wasmtime-go</code> repository</a></li>
+</ul>
+</div>
+</section>
+<section>
+<div class="container w-container">
+<h2>Documentation</h2>
+<p>The <a href="https://bytecodealliance.github.io/wasmtime">wasmtime guide</a> is the best starting point to learn about what
+Wasmtime can do for you or help answer your questions about Wasmtime. If you're
+curious in contributing to Wasmtime, <a href="https://bytecodealliance.github.io/wasmtime/contributing.html">it can also help you do
```

Comment:
```suggestion
interested in contributing to Wasmtime, <a href="https://bytecodealliance.github.io/wasmtime/contributing.html">it can also help you do
```